src/integrationtests/java/com/aws/greengrass/integrationtests/logmanager/LogManagerTest.java

```diff
@@ -283,4 +291,60 @@ void GIVEN_system_config_with_small_periodic_interval_WHEN_interval_elapses_THEN
         }
         assertEquals(1, allFiles.size());
     }
-}
+
+    @Test
+    void GIVEN_component_configs_WHEN_individual_configs_are_reset_THEN_correct_default_values_are_used()
+            throws Exception {
+        Path LOG_FILE_DIRECTORY_PATH_DEFAULT = tempRootDir.resolve("logs");
+
+        tempDirectoryPath = Files.createTempDirectory(tempRootDir, "IntegrationTestsTemporaryLogFiles");
+        setupKernel(tempDirectoryPath, "configsDifferentFromDefaults.yaml");
+        TimeUnit.SECONDS.sleep(30);
```

Comment:
why sleep 30? Is there a way to execute this test without it taking so long

Reply:
Thought I needed wait time after launching the kernel since that's what other tests do. Seem to work consistently without it though, so will remove the wait time.